deploy_and_run_tests.sh

```diff
@@ -224,11 +226,26 @@ if [[ -z "${skip_build}" ]]; then
       --substitutions _CUSTOM_TAG_NAME="${image_tag}" .
 fi
 
-source /opt/gcp_variant_transforms/venv3/bin/activate;
+# Running integration tests in a temporary virtualenv
+temp_dir="$(mktemp -d)"	
+color_print "Setting up integration test environment in ${temp_dir}" "${GREEN}"
+# Since we have no prompt we need to disable prompt changing in virtualenv.
+export VIRTUAL_ENV_DISABLE_PROMPT="something"
```

Comment:
It seems this was another hack for old venv, do we still need this?

Reply:
As per offline conversation, this is used to disable the prompt tag.
https://stackoverflow.com/questions/10406926/how-do-i-change-the-default-virtualenv-prompt/20026992